ddcz/models/used/creations.py

```diff
@@ -20,9 +20,12 @@
 logger = logging.getLogger(__name__)
 
 
+PUBLISHED = "a"
```

Comment:
Could you please now use `ddcz.creations.ApprovalChoices` instead? :)

Reply:
I am not sure I understand what you mean by use it instead. Could you provide me an example pls :)

Reply:
```
APPROVAL_CHOICES = (
    (ApprovalChoices.APPROVED.value, "Schváleno"),
    (ApprovalChoices.WAITING.value, "Neschváleno"),
)
```

Reply:
rebase on top of the current master to have it available

Reply:
Ah, thanks, I get it now. :) Hope now it's fine.